Common/include/option_structure.hpp

```diff
@@ -84,56 +84,56 @@ const unsigned int NO_RK_ITER = 0;            /*!< \brief No Runge-Kutta iterati
-const unsigned short COMM_TYPE_LONG           = 2;  /*!< \brief Communication type for long. */
-const unsigned short COMM_TYPE_UNSIGNED_SHORT = 3;  /*!< \brief Communication type for unsigned short. */
-const unsigned short COMM_TYPE_DOUBLE         = 4;  /*!< \brief Communication type for double. */
-const unsigned short COMM_TYPE_CHAR           = 5;  /*!< \brief Communication type for char. */
-const unsigned short COMM_TYPE_SHORT          = 6;  /*!< \brief Communication type for short. */
-const unsigned short COMM_TYPE_INT            = 7;  /*!< \brief Communication type for int. */
+const unsigned short COMM_TYPE_UNSIGNED_LONG = 1;  /*!< \brief Communication type for unsigned long. */
+const unsigned short COMM_TYPE_LONG = 2;           /*!< \brief Communication type for long. */
+const unsigned short COMM_TYPE_UNSIGNED_SHORT = 3; /*!< \brief Communication type for unsigned short. */
+const unsigned short COMM_TYPE_DOUBLE = 4;         /*!< \brief Communication type for double. */
+const unsigned short COMM_TYPE_CHAR = 5;           /*!< \brief Communication type for char. */
+const unsigned short COMM_TYPE_SHORT = 6;          /*!< \brief Communication type for short. */
+const unsigned short COMM_TYPE_INT = 7;            /*!< \brief Communication type for int. */
 
 /*!
  * \brief Types of geometric entities based on VTK nomenclature
  */
 enum GEO_TYPE {
-  VERTEX = 1,         /*!< \brief VTK nomenclature for defining a vertex element. */
-  LINE = 3,           /*!< \brief VTK nomenclature for defining a line element. */
-  TRIANGLE = 5,       /*!< \brief VTK nomenclature for defining a triangle element. */
-  QUADRILATERAL = 9,  /*!< \brief VTK nomenclature for defining a quadrilateral element. */
-  TETRAHEDRON = 10,   /*!< \brief VTK nomenclature for defining a tetrahedron element. */
-  HEXAHEDRON = 12,    /*!< \brief VTK nomenclature for defining a hexahedron element. */
-  PRISM = 13,         /*!< \brief VTK nomenclature for defining a prism element. */
-  PYRAMID = 14        /*!< \brief VTK nomenclature for defining a pyramid element. */
+  VERTEX = 1,        /*!< \brief VTK nomenclature for defining a vertex element. */
+  LINE = 3,          /*!< \brief VTK nomenclature for defining a line element. */
+  TRIANGLE = 5,      /*!< \brief VTK nomenclature for defining a triangle element. */
+  QUADRILATERAL = 9, /*!< \brief VTK nomenclature for defining a quadrilateral element. */
+  TETRAHEDRON = 10,  /*!< \brief VTK nomenclature for defining a tetrahedron element. */
+  HEXAHEDRON = 12,   /*!< \brief VTK nomenclature for defining a hexahedron element. */
+  PRISM = 13,        /*!< \brief VTK nomenclature for defining a prism element. */
+  PYRAMID = 14       /*!< \brief VTK nomenclature for defining a pyramid element. */
 };
-constexpr unsigned short N_ELEM_TYPES = 7;           /*!< \brief General output & CGNS defines. */
+constexpr unsigned short N_ELEM_TYPES = 7; /*!< \brief General output & CGNS defines. */
```

Comment:
Ok to start please revert the large formatting changes, I can't tell what you are adding to these files when everything is changed due to formatting :shrug: you know better what was added so please revert and re-apply only what is new.